pypeit/par/pypeitpar.py

```
@@ -990,7 +994,99 @@ def from_dict(cls, cfg):
+                          'Options are: {0}'.format(', '.join(options['method'])) + '.' + \
+                          '\'model\' will the scattered light model parameters derived from a ' \
+                          'user-specified frame during their reduction (note, you will need to make sure ' \
+                          'that you set appropriate scattlight frames in your .pypeit file for this option). ' \
+                          '\'frame\' will use each individual frame to determine the scattered light ' \
+                          'that affects this frame. ' \
+                          '\'archive\' will use an archival model parameter solution for the scattered ' \
+                          'light (note that this option is not currently available for all spectrographs).'
+
+        defaults['finecorr'] = True
+        dtypes['finecorr'] = bool
+        descr['finecorr'] = 'If True, a fine correction to the scattered light will be performed. However, the ' \
+                            'fine correction will only be applied if the model/frame/archive correction is performed.'
+
+        defaults['finecorr_pad'] = 2
+        dtypes['finecorr_pad'] = int
+        descr['finecorr_pad'] = 'Number of unbinned pixels to extend the slit edges by when masking the slits for the' \
+                                'fine correction to the scattered light.'
+
+        defaults['finecorr_order'] = 2
+        dtypes['finecorr_order'] = int
+        descr['finecorr_order'] = 'Polynomial order to use for the fine correction to the scattered light ' \
+                                  'subtraction. It should be a low value.'
+
+        defaults['finecorr_mask'] = None
+        dtypes['finecorr_mask'] = [int, list]
+        descr['finecorr_mask'] = 'A list containing the inter-slit regions that the user wishes to mask during ' \
+                                 'the fine correction to the scattered light. Each integer corresponds to an ' \
```

Just mention that the defaults mean no additional masking.

---

Done.